README.md

```diff
@@ -13,9 +13,12 @@ This app is presently under active development and we welcome contributions. Ple
 
 ## Installation Instructions
 
-Setting up Fabrik on your local machine is really easy. You can setup Fabrik using two methods:
+Setting up Fabrik on your local machine is very easy. You can setup Fabrik using two methods:
 
 ### Using Docker
+If you haven't installed Docker already: </br>
+[Use this link](https://docs.docker.com/docker-for-windows/install/) if you are using Windows. </br>
+[Use this link](https://docs.docker.com/docker-for-mac/install/) if you are using a Mac.
 
```

Comment:
@aaryangrg It still doesn't look neat, can you make it a little bit clean. You can refer any other lines from readme to make it clean.